service/controller/resource/ensurecpcrs/create.go

```diff
@@ -80,6 +81,40 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 }
 
 func (r *Resource) createAWSControlPlaneCR(ctx context.Context, cr infrastructurev1alpha2.AWSCluster, id string) error {
+	cp := &infrastructurev1alpha2.AWSControlPlane{
+		TypeMeta: infrastructurev1alpha2.NewAWSControlPlaneTypeMeta(),
+		ObjectMeta: metav1.ObjectMeta{
+			Annotations: map[string]string{
+				annotation.Docs: "https://godoc.org/github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2#AWSControlPlane",
```

Comment:
@marians I reckon there is no better URL to link to ATM. We can update these later but I think the godoc is a good place to link to just to begin with.